bookdown/02-basics.Rmd

```diff
@@ -38,7 +38,7 @@ We cover how to
 
 [**Learners**](#learners)
 
-[Learners](#learners) encapsulate machine learning algorithms to train models and make predictions for a `r ref("mlr3::Task")`.
+[Learners](#learners) encapsulate machine learning algorithms to train models and make predictions for a [task](#tasks).
```

Comment:
changed bc. in this chapter there are no technical refs and this looked strange.